src/com/puzzleitc/jenkins/command/GitCryptCommand.groovy

```diff
@@ -0,0 +1,30 @@
+package com.puzzleitc.jenkins.command
+
+import com.puzzleitc.jenkins.command.context.PipelineContext
+
+class GitCryptCommand {
+
+    private final PipelineContext ctx
+    private String credentialsId
+
+    GitCryptCommand(PipelineContext ctx) {
+        this.ctx = ctx
+    }
+
+    void execute() {
+        credentialsId = ctx.stepParams.getOptional('credentialsId', null) as String
```

Comment:
Beim KustomizeCommand haben wir auch nur einen einzelnen Parameter und den geben wir direkt mit und nicht in der Param-Map. Ich finde es aber so fast besser, weil wir dann alle Steps gleich handhaben können und weil der Command leichter erweiterbar ist.

Ich würde also dementsprechend den KustomizeCommand noch angleichen.

Was meinst du?